PWGGA/Hyperon/AliSigma0PhotonMotherCuts.cxx

```diff
@@ -285,12 +291,15 @@ void AliSigma0PhotonMotherCuts::SigmaToLambdaGamma(
 
       if (std::abs(rap) > fRapidityMax) continue;
       const int rapBin = GetRapidityBin(rap);
+      const int multBin = GetMultiplicityBin(lPercentile);
       if (!fIsLightweight) {
```

Comment:
The histograms are initialised anyways, so could you fill them all the time (not only when fIsLightweight is true
Just move them to where fHistInvMassPt is filled (and accordingly for the others)